public/index.html

```diff
@@ -6,6 +6,8 @@
     <title>Static list of goods</title>
   </head>
   <body>
-    <div id="root"></div>
+    <div id="root">
+      
+    </div>
```

Comment:
```suggestion
    <div id="root"></div>
```
There is no need to change anything here